tests/wallet/cat_wallet/test_offer_lifecycle.py

```diff
@@ -297,15 +254,16 @@ async def test_complex_offer(cost_logger):
         assert new_offer.is_valid()
 
         # Test preventing TAIL from running during exchange
-        blue_cat_puz: Program = construct_cat_puzzle(CAT_MOD, str_to_tail_hash("blue"), OFFER_MOD)
-        blue_spend: CoinSpend = make_spend(
-            Coin(bytes32(32), blue_cat_puz.get_tree_hash(), uint64(0)),
+        blue_cat_puz = construct_cat_puzzle(CAT_MOD, str_to_tail_hash("blue"), OFFER_MOD)
+        random_hash = bytes32([0] * 32)
```

Comment:
There's `bytes32.random()`, and if you use the `seeded_random: Random` fixture then you can `bytes32.random(r=seeded_random)` for deterministic random `bytes32`s.  But sure, `bytes32(32)` is also valid for just zeros.

Reply:
Thanks. `bytes32(32)` is currently problematic with `mypy` and so I went with this in consistency with that code.